clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -220,12 +223,12 @@ static void ParseArgs(int argc, char **argv) {
   InMemoryCAS = Args.hasArg(OPT_in_memory_cas);
 
   if (const llvm::opt::Arg *A = Args.getLastArg(OPT_cas_path_EQ))
-    CASOpts.CASPath = A->getValue();
+    CASConfig.CASPath = A->getValue();
```

Comment:
Here, we can just parse into a `static std::string CASPath`, then construct the CASOptions in the actual invocation?

Reply:
Then we can simply follow the rule that word `CAS` should not appear in global/static variables types.

Reply:
Done in 033c965